### PR TITLE
Implement delete role

### PR DESCRIPTION
**Pull request by hothanhtuan1209**
#13

### CODE REVIEW

**Comment by hothanhtuan1209** [comment]
unnamed
unnamed (1)

File: myproject/supermarket_manager/urls.py [hunk 1]
```diff
@@ -1,8 +1,14 @@
 from django.urls import path
-from .views import add_role, list_roles, update_role
+from .views import (
+    add_role,
+    list_roles,
```
Comment (hungle-agilityio) [inline myproject/supermarket_manager/urls.py:4]: Using `get_roles` instead if `list_roles`

File: myproject/supermarket_manager/views.py [hunk 2]
```diff
@@ -93,3 +94,28 @@ def update_role(request, role_id):
...
+        role_id (str): The ID of the role to be deleted.
+
+    Returns:
+        JsonResponse: A JSON response indicating the result of the delete operation.
+    """
+    if request.method == "DELETE":
+        try:
+            role = Role.objects.get(role_id=role_id)
+            role.delete()
+            return JsonResponse({"message": DELETED}, status=200)
```
Comment (hungle-agilityio) [inline myproject/supermarket_manager/views.py:117]: Please double check about status when delete success.

File: myproject/supermarket_manager/urls.py (hunk 1 above)
Comment (hothanhtuan1209) [reply myproject/supermarket_manager/urls.py:4]: Done

File: myproject/supermarket_manager/views.py (hunk 2 above)
Comment (hothanhtuan1209) [reply myproject/supermarket_manager/views.py]: Done

**Pull request merged**